docs-src/concepts/what-is-asynchronous-activity-completion.md

```diff
@@ -19,11 +19,20 @@ The Temporal Client can then be used to both Heartbeat Activity Execution progre
 
 #### When to use Async Completion
 
-The intended use-case for this feature is when an external system has the final result of a computation, started by an Activity.
+The intended use case for this feature is when an external system has the final result of a computation, started by an Activity.
+However, often it works better for the Activity to notify the external system, complete normally, and then wait in the Workflow for a [Signal](/concepts/what-is-a-signal) from the external system.
 
-Consider using Asynchronous Activities instead of Signals if the external process is unreliable and might fail to send critical status updates through a Signal.
+Consider using Async Completion when
 
-Consider using [Signals](/concepts/what-is-a-signal) as an alternative to Asynchronous Activities to return data back to a Workflow Execution if there is a human in the process loop.
-The reason is that a human in the loop means multiple steps in the process.
-The first is the Activity Function that stores state in an external system and at least one other step where a human would “complete” the activity.
-If the first step fails, you want to detect that quickly and retry instead of waiting for the entire process, which could be significantly longer when humans are involved.
+- the external process is unreliable and might fail to send critical status updates through a Signal.
+- you want the external process to Heartbeat or receive Cancellation.
+
+Consider using Signals when you want to retry sooner a failure to notify the external system.
+
+Immediate retries are helpful when the external process might take a long time.
+For example, consider an external process that is waiting for a human to review something and respond, and might take days.
+If you set a [Start-To-Close Timeout](/activities#start-to-close-timeout) of one week on an Activity with Async Completion, the Activity might fail to notify the external system and not throw an error (for example, if the Worker dies).
```

Comment:
Notify the external system of what?
I think this example depends on a mapped out information flow -
ATM I find it hard to keep track of that model in my head when I read it.

Would you be willing to completely lay out the scenario from the perspective of both so that it is easier to compare?


Also - it sounds like Async completion has a very limited use case - Is there one we can call out explicitly here?